Basic-Car-Maintenance/Documentation.docc/Documentation.md

```diff
@@ -16,3 +16,76 @@ Icons have the following colors:
 * Blue #253AA7
 * Black #000000
 * Orange #F78616
+
+## Firestore Collections
+
+**_alerts_**  
+The alerts collection contains system level alerts that will be visable to all users
+
+| Fields    | Type    |
```

Comment:
Can you add a description too, for that that property does? thanks!

Reply:
What's the difference between actionText and message. What is actionURL for?

Reply:
The `actionText` is the text for the button, the `actionURL` is url that the user will go to when they tap the button